_data/sarthakgupta072/sarthakgupta072.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Sarthak Gupta
+institution: Birla Institute of Technology, Mesra 🚩
+profile_pic: sarthakgupta072.png
```

Comment:
Image file path name seems incorrect, please change it to `sarthakgupta072.jpg `